style.css

```diff
@@ -118,21 +137,127 @@ p {
   display: flex;
   margin-top: 12px;
   padding-left: 0;
-  justify-content: flex-start;
+  justify-content: start;
+  align-items: flex-start;
   gap: 20px;
 }
 
+.contact {
+  background-image: url("./images/contact\ form\ shapes-mobile.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-radius: 100px 0 0 0;
+  padding-top: 114px;
+  padding-bottom: 114px;
+  padding-left: 24px;
+  padding-right: 24px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.first-text-contact {
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+  align-self: flex-start;
+  padding-bottom: 20px;
+}
+
+.second-text-contact {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  color: #ebebff;
+  text-align: center;
+  align-self: flex-start;
+  margin: 18px 0;
+}
+
+.text-area {
+  font-family: Poppins, sans-serif;
+  padding: 12px 20px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  background-color: rgb(255, 255, 255);
+  background-color: #f8f8f8;
+  resize: none;
+  margin-bottom: 1px;
+  width: 100%;
+  align-self: flex-start;
+  padding-left: 20px;
+  outline: none;
+}
+
+.text-area-large {
+  font-family: Poppins, sans-serif;
+  width: 100%;
+  height: 170px;
+  padding: 12px 20px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  background-color: rgb(255, 255, 255);
+  background-color: #f8f8f8;
+  resize: none;
+  margin-bottom: 18px;
+  align-self: flex-start;
+  outline: none;
+}
+
+.btn-container-contact {
+  text-align: left;
+  border-radius: 8px;
+  border: none;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  cursor: pointer;
+  padding: 8px;
+  align-self: center;
+}
+
+.btn-container-contact:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.erro {
+  width: 100%;
+  text-align: center;
+  align-content: center;
+  font-size: 80%;
+  color: white;
+  background-color: rgb(247, 70, 70);
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.container-form {
+  display: flex;
+  row-gap: 18px;
+  width: 100%;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 30px 20%;
+}
+
 .flex-center-culomn {
   display: flex;
   justify-content: center;
   align-items: center;
   flex-direction: column;
+  padding: 65px 0;
 }
 
 .card-item {
   border: 1px solid #7a869a;
```

Comment:
- [ ] Kindly add the right border color `#DFE1E6` for each card just as the previous [reviewer requested](https://github.com/BouaikIlham/Portfolio-project/pull/4#discussion_r840352359)